src/GLideNUI/GLideNUI.cpp

```diff
@@ -96,6 +81,25 @@ int runAboutThread(const wchar_t * _strFileName) {
 	return 0;
 }
 
+EXPORT void CALL ConfigInit(void * hinst)
+{
+	initMyResource();
+
+	pAppInstance = QCoreApplication::instance();
+
+	if (pAppInstance == nullptr) {
+		int argc = 0;
+		char * argv = 0;
+		pQApp.reset(new QApplication(argc, &argv));
+		pAppInstance = pQApp.get();
+	}
+}
+
+EXPORT void CALL ConfigCleanup(void)
+{
+	cleanMyResource();
```

Comment:
Application crashes on exit. now
I think, pQApp should be released here.

Reply:
can you re-test with the latest push? I personally haven't experienced any crashes myself.

Reply:
This has fixed the Pj64 plugins screen crashing too when not the selected plugin.